inference-engine/src/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -136,6 +137,7 @@ bool ngraph::pass::MOCTransformations::run_on_function(std::shared_ptr<ngraph::F
     auto decomp = manager.register_pass<ngraph::pass::GraphRewrite>();
     decomp->add_matcher<ngraph::pass::BatchNormDecomposition>();
     decomp->add_matcher<ngraph::pass::ConvertDivideWithConstant>();
+    decomp->add_matcher<ngraph::pass::ConvertSubtract>();
```

Comment:
We can't just enable Subtract decomposition as is because it can affect dequantization sub-graph. So first we need to markup all special Subtract operation and only then we can enable this.